d2/src/main/java/com/linkedin/d2/xds/XdsClientImpl.java

```diff
@@ -77,6 +77,7 @@ public class XdsClientImpl extends XdsClient
       Arrays.stream(ResourceType.values())
           .filter(e -> e.typeUrl() != null)
           .collect(Collectors.toMap(Function.identity(), e -> new HashMap<>())));
+  private final Map<ResourceType, WildcardResourceSubscriber> _wildcardSubscribers = new HashMap<>();
```

Comment:
Nit: I think you could use `Maps.newEnumMap` here to match the above map.

Reply:
Good idea. Updated